sonic_api.yaml

```diff
@@ -491,6 +491,39 @@ paths:
           description: Maintanence mode
           schema:
             $ref: '#/definitions/Error'
+    '/config/interface/vlans/all':
```

Comment:
Do we need the 'all' here? We don't seem to have any other api ending with "vlans".

Reply:
Yes, we need "all". There is other api ending with "vlans" for those specific to vnet.